orchagent/vxlanorch.h

```diff
@@ -190,11 +203,13 @@ class VxlanTunnel
     // Total Routes using the DIP tunnel. 
     int getDipTunnelRefCnt(const std::string);
     int getDipTunnelIMRRefCnt(const std::string);
-    int getDipTunnelIPRefCnt(const std::string);
+    int getRemoteEndPointIPRefCnt(const std::string);
```

Comment:
Sure I understand that we would like a naming common for P2P and P2MP.  So it might be better to change the naming for the IMR as well as the total refcnt i.e. we can eliminate the use of DipTunnel in the name for all the common cases.

Reply:
Modified as suggested.